### DISCUSSION TITLE
Instant-NGP 설치하는데 있어 cmake . -B build가 잘 되지 않아 궁금해서 올리게 되었습니다.

### DISCUSSION THREAD

**Discussion by simin4** [NVlabs/instant-ngp#1297] [General]
안녕하세요! Instant-NGP 설치 문제가 막혀 올리게 되었습니다.
https://www.youtube.com/watch?v=kq9xlvz73Rg 이 영상을 보며 설치 하고 있습니다.
9분 20초 cmake . -B build를 복사해서 넣는데 진행이 되지 않습니다.

 D드라이브에 Tutorial라는 폴더를 만들고, 그 안에 ngp라는 폴더를 만들어 instant-ngp를 설치했습니다.

밑에 처럼 작성을 했지만 'cmake'은(는) 내부 또는 외부 명령, 실행할 수 있는 프로그램, 또는
배치 파일이 아닙니다. 라는 결과만 나옵니다.

*************************************************

C:\Program Files (x86)\Microsoft Visual Studio\2019\Community>cd D:\Tutorial\ngp\instant-ngp

C:\Program Files (x86)\Microsoft Visual Studio\2019\Community>D:

D:\Tutorial\ngp\instant-ngp>cmake . -B build
'cmake'은(는) 내부 또는 외부 명령, 실행할 수 있는 프로그램, 또는
배치 파일이 아닙니다.

**Comment by alibeddiaf** [comment]
Hi,
I faced this issue before, and I fixed it.
It appears that I had many cmake commands installed on my machine (with different versions and vendors) and inserted in the path env variable.

Ensure that the one executed is the one that comes from MS visual studio community 2019 for instance instead of writing the command cmake . -B build Write the full path of the command : C:\Program Files (x86)\Microsoft Visual Studio\2019\Community...\cmake.exe . -B build

Good luck
